service/application/src/main/java/com/solace/maas/ep/event/management/agent/command/SempDeleteCommandManager.java

```diff
@@ -0,0 +1,271 @@
+package com.solace.maas.ep.event.management.agent.command;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.solace.client.sempv2.ApiException;
+import com.solace.client.sempv2.api.AclProfileApi;
+import com.solace.client.sempv2.api.AuthorizationGroupApi;
+import com.solace.client.sempv2.api.ClientUsernameApi;
+import com.solace.client.sempv2.api.QueueApi;
+import com.solace.maas.ep.common.model.SempAclProfileDeletionRequest;
+import com.solace.maas.ep.common.model.SempAclPublishTopicExceptionDeletionRequest;
+import com.solace.maas.ep.common.model.SempAclSubscribeTopicExceptionDeletionRequest;
+import com.solace.maas.ep.common.model.SempAuthorizationGroupDeletionRequest;
+import com.solace.maas.ep.common.model.SempClientUsernameDeletionRequest;
+import com.solace.maas.ep.common.model.SempEntityType;
+import com.solace.maas.ep.common.model.SempQueueDeletionRequest;
+import com.solace.maas.ep.common.model.SempQueueTopicSubscriptionDeletionRequest;
+import com.solace.maas.ep.event.management.agent.command.semp.SempApiProvider;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.Command;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.CommandResult;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.CommandType;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.JobStatus;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.SempDeleteCommandConstants;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang.Validate;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+import static com.solace.maas.ep.event.management.agent.plugin.command.model.SempDeleteCommandConstants.SEMP_DELETE_DATA;
+import static com.solace.maas.ep.event.management.agent.plugin.terraform.manager.TerraformUtils.setCommandError;
+
+@Service
+@Slf4j
+public class SempDeleteCommandManager {
+
+    public static final String ERROR_EXECUTING_COMMAND = "Error executing command";
+
+    private final ObjectMapper objectMapper;
+
+    public SempDeleteCommandManager(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    public void execute(Command command, SempApiProvider sempApiProvider) {
+        try {
+            validate(command, sempApiProvider);
+            executeSempDeleteCommand(command, sempApiProvider);
+            command.setResult(CommandResult.builder()
+                    .status(JobStatus.success)
+                    .logs(List.of())
+                    .build());
+        } catch(ApiException e) {
+            ApiException apiException = (ApiException) e;
```

Comment:
On second look, basically we can consolidate both exception 
```
} catch (Exception e) {
    if (e instanceof ApiException) {
        ApiException apiException = (ApiException) e;
        log.error("Error executing SEMP delete command: {}", apiException.getResponseBody());
    } else {
        log.error(ERROR_EXECUTING_COMMAND, e);
    }
    setCommandError(command, e);
}
```

Reply:
I also have a concern with this log `log.error("Error executing SEMP delete command: {}", apiException.getResponseBody());`
It can potentially leak sensitive information (such as client username) in plain text